### ISSUE TITLE
Unable to view ICONS on IOS Device

### ISSUE THREAD

**Issue by rksambari** [google/model-viewer#3935]
Hi,

I have added a few ICONS in my code and they are working perfectly fine when I open with an Android device, but those ICONS are not visible when opened with IOS Device.
FYI, I have used AR Modes **webxr** and **quick-look**.
Please refer to the attached image(ICONS right side, youtube icons).
![AR](https://user-images.githubusercontent.com/99704624/201016206-fd5f0995-f5cf-4a23-949a-c6bc53f972d4.jpeg)

**Comment by elalish** [comment]
Indeed, please petition Apple to support WebXR so iOS can have as good an AR experience as Android (@grorg).
